typescript/sdk/package.json

```diff
@@ -17,7 +17,7 @@
   "devDependencies": {
     "@nomiclabs/hardhat-ethers": "^2.2.1",
     "@nomiclabs/hardhat-waffle": "^2.0.3",
-    "@types/node": "^16.9.1",
+    "@types/node": "^18.0.0",
```

Comment:
Was there something in node 18's core lib that we need? If so we may need to add a polyfill for it to keep the SDK working in the browser